#### What is variant?

- Genetic variants are differences in the DNA sequences of individuals within a population.
- Genetic variations are changes at the nucleotide level, it can be single or more base changes, difference in the number of copies of DNA segments or even a partial/full chromosome.
- These variations cause diversity among individuals in genotype and phenotype.
- These genetic variations can be further interpreted based on their consequences on mRNA and protein level.

 Genetic variants can be

- Somatic: Somatic variants arise during an individual's lifetime in tissues other than germ cells and are not passed to subsequent generations.

- Germline: Germline variants occur in all cells including germ cells and inherited from parents or arise spontaneously as de novo.

#### Types of genetic variants

DNA variants 
   1. Nucleotide level:
         - Single nucleotide variants
         - Small indels : Insertions, Deletions, Substitutions
   2. Chromosomal level:
         - Aneuploidies
         - Structural variants : Transloactions, Repeats, Inversions
         - Copy number variants: Amplifications (gain), Deletions (loss)


- Small Indels: short insertion-deletions of nucleotides with in DNA sequence, typically ranges from 1-50 bases 

- SNV : Changes at a single nucleotide within DNA sequence, it can be substitution ( Replacement of one nucleotide with another), deletion (Removal of single nucleotide) and insertion (Addition of a single nucleotide)

- Aneuploidy: Abnormality in the number of chromosomes 

- Large structutal variants: Alteration that cause changes in the structure, orgnaisation or arrangement of larger DNA segments.

    a. Inversions: Genetic rearrangement where a segment of DNA is reversed in orientation within the chromosome with two breakpoints.

    b. Translocation: Rearrangement of genetic material between non-homologous chromosomes

    c. Repeat expansions: Increase in the number of repeating DNA sequence within a gene.

- Copy number variations (CNV): Genetic alteration that involve the duplication or deletion of segments of DNA (>50 bases), resulting in changes in copy number of specific regions of the genome.

   a. Amplification: increase in the number of copies of a specific genetic sequence within the DNA. 

   b. Deletion: Loss of a portion of a DNA segment, resulting in fewer copies of specific genetic sequence.


#### Variant effect on mRNA

DNA sequence of a gene are transcribed to produce mRNA and the codons in the mRNA are transalated into a polypeptide chain. 

RNA variations

- RNA substitution
  
  -  Missense/Nonsynonymous variants : Variation that changes one amino acid to naother in a portein.

  -  Nonsense/stop gain variants: Variation that converts regulat codon into a stop codon, leading to premature protein termination.

  -  Synonymous/Silent variants : Variation that doesn't alter the encoded aino acid, mostly occur at 3rd position of the codon, s these nucleotides are redundant.

  -  Start loss varinats : Variation prevents the  transalation intiation at the usual start codon 

  -  Stop loss variants: Variation that prevents the termination of transalation, resulting in longer protein.

- RNA deletion
  
  - In-frame deletion : Deletion of nucleotide that maintains the reading frame of a gene, usually three or multiple of three nucleotides are changed.

  - Frameshift deletions: Deletion of nucleotides that alter the reading frame, often causing nonfunctional protein.

- RNA insertion
  
  - In-frame insertions : Addition of nucleotides that maintain the reading frame of gene, usually three or multiple of three nucleotides are changed.

  - Frameshift insertions : Addition of nucleotide that disrupts the reading frame, typically leading to nonfunctional protein.

  #### Variant nomeclature

  - standard used to describe the variations found in DNA, RNA, Protein , which is set by Human Genome Variation Society (HGVS). 
  - The standard comprises of three elements
     - Reference sequence
     - Description of variant, location in coding sequence or genome 
     - Predicted consequences 
Example: NM-004006.2:c.4375C>T p.(Arg1459*), results in truncated nonfunctional protein.

- NM-004006.2: Refernce sequence in Genbank,"NM" for mRNA sequence, "NR" for non-coding RNA sequence.
- c.4375C>T : Describe variant, indicating change at the nucleotide level, specifically at the coding sequence at position 4375 from C(cytosine) to a T(Thymine). "c" is the coding DNA refernce sequence, "g" is geneome reference sequence, "n" non-coding regions.
- p.(Arg1459*) : Predicted effect at the protein level as being stop codon (*) at position 1459 in the arginine.

#### Check the variant validity

https://www.variantvalidator.org/

#### Variant Interpretation

- When change occurs in the genome,it cause an impact on the function of the cell and in some cases can lead to disease.

- Genetic variant interpretation use various sources of evidences to decide whether the chnage can be disease causing or not. Sometimes there won't be enough evidences and sometimes the disease is complex to make it difficult to interpret the variant.

- NGS has made it possible to understand the variantions in the genome all at once.

- Process of variant interpretation includes

    a. Variant annotation : 
    
    Process of adding information about the variant or a gene. For eg. Gene Nomenclature and consequences. More inforation withh be helpful to interpret the potential impact of a variant.

    b. Variant prioritisation : Gene-disease association, Variant type, Sequence quality, Population frequency(Common vaiants are unlikely to cause a rare disease), Inheritance pattern (Two pathogenic alleles should be present in a recessive disorder, only one pathogenic allele will lead to a disease state in dominany disorder), potential impact ( genetic variant leads to protein change and disruption of function).

    c. Variant classification and interpretation :
    Whether the variant support pathogenicity is checked based on standardised guidelines for variant interpretation issued by The American College of Medical Genetics and Genomics (ACMG) and whether the classification matches the clinical feature?
